WindowsDevicePortalWrapper/WindowsDevicePortalWrapper/DefaultDevicePortalConnection.cs

```diff
@@ -45,6 +44,29 @@ public class DefaultDevicePortalConnection : IDevicePortalConnection
         }
 
         /// <summary>
+        /// Initializes a new instance of the <see cref="XboxDevicePortalConnection"/> class, using a SecureString to secure the password.
```

Comment:
Looks like your rename to DefaultDevicePortalConnection got lost here. Also, I'll close the other PR that did this since that one is out-of-date.